web/src/app/validation/process/decision-multiple-outgoing-connections-validator.ts

```diff
@@ -0,0 +1,24 @@
+import { ElementValidatorBase } from '../element-validator-base';
+import { Process } from '../../model/Process';
+import { Validator } from '../validator-decorator';
+import { IContainer } from '../../model/IContainer';
+import { ValidationResult } from '../validation-result';
+import { Type } from '../../util/type';
+import { Config } from '../../config/config';
+import { ProcessStep } from '../../model/ProcessStep';
+import { ProcessDecision } from '../../model/ProcessDecision';
+import { ProcessConnection } from '../../model/ProcessConnection';
+
+@Validator(Process)
+export class DecisionMultipleOutgoingConnectionsValidator extends ElementValidatorBase<Process> {
+    public validate(element: Process, contents: IContainer[]): ValidationResult {
+        const invalidNodes: IContainer[] = contents
+            .filter((element: IContainer) => Type.is(element, ProcessDecision))
+            .filter((element: ProcessDecision) =>
+                element.outgoingConnections && element.outgoingConnections.length < 2);
```

Comment:
what is if element.outgoingConnection == undefined? In this case there would be no outgoing connections (which means the decision node is invalid) BUT the validator would not flag the decision node as invalid.

Reply:
This should be covered by a different check: node-no-outgoing-validator.ts. There, we flag all node, that do not have outgoind nodes as invalid. (Of course, not end nodes)